src/handle.c

```diff
@@ -70,7 +70,7 @@ static void luv_close_cb(uv_handle_t* handle) {
 static int luv_close(lua_State* L) {
   uv_handle_t* handle = luv_check_handle(L, 1);
   if (uv_is_closing(handle)) {
-    luaL_error(L, "handle %p is already closing", handle);
+    return luaL_error(L, "handle %p is already closing", handle);
```

Comment:
Does this suppress a compiler warning?  `luaL_error` never returns since it does a `longjmp` as part of the exception process.